### DISCUSSION TITLE
https://blog.3place.cn/3place/lmyn/LMYN050.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#99] [General]
# https://blog.3place.cn/3place/lmyn/LMYN050.html

https://blog.3place.cn/3place/lmyn/LMYN050.html

**Comment by picklecai** [comment]
为了想看那张大弓，
他们又来到了此间。” （1.49.24）

这回众友说话倒直截了当，没有藏着掖着。

**Comment by picklecai** [comment]
进一步又把他来问： （1.49.16）
“这两位王子，愿你们有福！
勇敢可以比得上神仙，
步伐像是大象和狮子，
两位英雄像老虎、公牛一般〔392〕。 （1.49.17）
他俩的大眼睛像荷叶，
随身带着刀剑、箭壶和弓；
样子长得像那阿湿波，
他俩正是在青春妙龄〔393〕。 （1.49.18）
他俩像是天上的神仙，
偶然降落到了人间。
牟尼呀！他俩怎样用双脚走来？
是谁的孩子？有什么公干？〔394〕 （1.49.19）
两个英雄带着精良的兵器。
大牟尼呀！两个孩子是谁的？
他俩照亮了这个地方，
有如日月照耀天宇〔395〕。 （1.49.20）
他俩彼此是这样相像，
一举一动都循规蹈矩〔396〕。
两位英雄有着鬓角卷发。
我想听一个仔仔细细。”〔397〕 （1.49.21）
听完了高贵尊严的
遮那竭国王说的话，
他就把高贵尊严的
十车王二子介绍给他： （1.49.22）
“他俩住在‘完成’净修林，
曾经把罗刹们来杀伤〔398〕。

和12.7的内容一模一样。
12.7问候的人是须摩底。
12.9问候的人是须摩底。
